src/Umbraco.Web/Models/Mapping/MediaModelMapper.cs

```diff
@@ -65,6 +68,20 @@ public override void ConfigureMappings(IConfiguration config, ApplicationContext
                 .ForMember(dto => dto.Icon, expression => expression.Ignore())
                 .ForMember(dto => dto.Alias, expression => expression.Ignore())
                 .ForMember(dto => dto.HasPublishedVersion, expression => expression.Ignore());
+
+            //FROM ContentTypeBasic TO ContentPropertyDisplay
+            config.CreateMap<ContentTypeBasic, ContentPropertyDisplay>()

```

Comment:
Why do we need this?

Reply:
I have created a map to be able to Use ``MediaTypeBasicResolver`` in order to map the ContentType property for the MediaItem.

Reply:
Can you point me to where this is used/needed, just seems a bit odd

Reply:
Sure @Shazwazza , while using the ``MediaTypeBasicResolver`` and not have the map created, in ``MediaController`` I get a AutoMaper exception with the following message:

+		$exception	{"Missing type map configuration or unsupported mapping.\r\n\r\nMapping types:\r\nContentTypeBasic -> ContentPropertyDisplay\r\nUmbraco.Web.Models.ContentEditing.ContentTypeBasic -> Umbraco.Web.Models.ContentEditing.ContentPropertyDisplay\r\n\r\nDestination path:\r\nMediaItemDisplay.ContentType.ContentType\r\n\r\nSource value:\r\nUmbraco.Web.Models.ContentEditing.ContentTypeBasic"}	AutoMapper.AutoMapperMappingException

![capture](https://user-images.githubusercontent.com/14214114/31237343-f781af8e-a9f6-11e7-98fd-f3918ff04abb.PNG)

Following AutoMapper's instructions and creating the map fixed the issue, can this be done in a better way?

Reply:
So there's some odd and inconsistent things going on here:

Reply:
I have pushed changes for this, @RobertCopilau  please review that what I've changed makes sense and that the correct json is returned.

Reply:
Yes... I can see the mistake I have made. I have changed front-end to match the new values received and everything works as it should.